monitoring/angle-monitoring/src/main/java/com/farao_community/farao/monitoring/angle_monitoring/RedispatchAction.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.monitoring.angle_monitoring;
+
+import com.powsybl.glsk.api.GlskPoint;
+import com.powsybl.glsk.api.GlskRegisteredResource;
+import com.powsybl.glsk.api.GlskShiftKey;
+import com.powsybl.glsk.api.util.converters.GlskPointScalableConverter;
+import com.powsybl.iidm.network.Network;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import static com.farao_community.farao.commons.logs.FaraoLoggerProvider.BUSINESS_WARNS;
+
+/**
+ * A redispatch action collects a quantity of power to be redispatched (powerToBeRedispatched) in country (countryName)
+ * according to merit order glsks that exclude networkElementsToBeExcluded.
```

Comment:
I think it's generic (not merit order specific) since it relies on powsybl, no ?